apps/.storybook/config.js

```diff
@@ -103,6 +104,19 @@ storybook.setAddon({
   }
 });
 
+storybook.setAddon({
+  withReduxStore(reducers = {}) {
+    this.addDecorator(story => {
+      this.store = createStore(combineReducers(reducers));
+      return (
+        <Provider store={this.store}>
+          {story()}
+        </Provider>
+      );
+    });
+  }
```

Comment:
In the test storybook version you had to return `this` to make this method chainable.  That doesn't seem to be necessary here (through what mechanism I don't know) but maybe we should just for consistency?